bitnami/keycloak/templates/realms.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if not .Values.KcCasC.existingConfigSecret }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "keycloak-config-cli.fullname" . }}-config-realms
+  labels:
+  {{- include "keycloak-config-cli.labels" . | nindent 4 }}
+data:
+  {{- range $name, $config := .Values.config }}
+    {{- if hasKey $config "file" }}
+  {{ $name }}.json: "{{ tpl ($.Files.Get $config.file) $ | b64enc }}"
```

Comment:
We try not using the Files function as it caused issues in the past. We prefer using either the inline, the existingConfig or the extraDeploy value.